iapetus/iapetus.py

```diff
@@ -172,7 +171,10 @@ def run(self, platform_name=None, precision='auto', max_n_contexts=None):
             Precision to use, or None to automatically select ('mixed' is used if supported)
         max_n_contexts : int, optional, default=None
             Maximum number of contexts to use
-
+        resume : bool, default=False
```

Comment:
What's the difference between `resume` and `extend` here? What happens if both are passed as `True`? It looks like the code will favor `resume` over `extend`---we should document that if we're going to use this approach. Alternatively, we could have the simulation try to *automatically* resume if it already exists, and automatically extend by the specified number of iterations regardless.